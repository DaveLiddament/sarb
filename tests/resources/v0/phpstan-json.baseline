{"historyAnalyser":"git","historyMarker":"0f078b7d445f661991605b91d8ddb9c7f796242d","resultsParser":"phpstan-json-tmp","analysisResults":[{"lineNumber":89,"fileName":"static-analysis-baseliner\/src\/Domain\/BaseLiner\/BaseLineImporter.php","type":"Parameter #1 $array of static method expects int, array given.","message":"Parameter #1 $array of static method DaveLiddament\\StaticAnalysisResultsBaseliner\\Domain\\ResultsParser\\AnalysisResults::fromArray() expects int, array given.","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/BaseLiner\\\/BaseLineImporter.php\",\"message\":{\"message\":\"Parameter #1 $array of static method DaveLiddament\\\\StaticAnalysisResultsBaseliner\\\\Domain\\\\ResultsParser\\\\AnalysisResults::fromArray() expects int, array given.\",\"line\":89,\"ignorable\":true,\"relativePath\":\"Domain\\\/BaseLiner\\\/BaseLineImporter.php\"}}"},{"lineNumber":73,"fileName":"static-analysis-baseliner\/src\/Domain\/ResultsParser\/AnalysisResults.php","type":"PHPDoc tag @param for parameter $array with type array is incompatible with native type int","message":"PHPDoc tag @param for parameter $array with type array is incompatible with native type int","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/ResultsParser\\\/AnalysisResults.php\",\"message\":{\"message\":\"PHPDoc tag @param for parameter $array with type array is incompatible with native type int\",\"line\":73,\"ignorable\":true,\"relativePath\":\"Domain\\\/ResultsParser\\\/AnalysisResults.php\"}}"},{"lineNumber":0,"fileName":"static-analysis-baseliner\/src\/Domain\/ResultsParser\/AnalysisResults.php","type":"Argument of an invalid type int supplied for foreach, only iterables are supported.","message":"Argument of an invalid type int supplied for foreach, only iterables are supported.","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/static-analysis-baseliner\\\/src\\\/Domain\\\/ResultsParser\\\/AnalysisResults.php\",\"message\":{\"message\":\"Argument of an invalid type int supplied for foreach, only iterables are supported.\",\"line\":null,\"ignorable\":true,\"relativePath\":\"Domain\\\/ResultsParser\\\/AnalysisResults.php\"}}"}]}