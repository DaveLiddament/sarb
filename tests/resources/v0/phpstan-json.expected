{
    "historyAnalyser": "git",
    "historyMarker": "0f078b7d445f661991605b91d8ddb9c7f796242d",
    "resultsParser": "phpstan-json",
    "analysisResults": [
        {
            "lineNumber": 89,
            "fileName": "static-analysis-baseliner\/src\/Domain\/BaseLiner\/BaseLineImporter.php",
            "type": "Parameter #1 $array of static method expects int, array given.",
            "message": "Parameter #1 $array of static method DaveLiddament\\StaticAnalysisResultsBaseliner\\Domain\\ResultsParser\\AnalysisResults::fromArray() expects int, array given.",
            "severity": "error"
        },
        {
            "lineNumber": 73,
            "fileName": "static-analysis-baseliner\/src\/Domain\/ResultsParser\/AnalysisResults.php",
            "type": "PHPDoc tag @param for parameter $array with type array is incompatible with native type int",
            "message": "PHPDoc tag @param for parameter $array with type array is incompatible with native type int",
            "severity": "error"
        },
        {
            "lineNumber": 0,
            "fileName": "static-analysis-baseliner\/src\/Domain\/ResultsParser\/AnalysisResults.php",
            "type": "Argument of an invalid type int supplied for foreach, only iterables are supported.",
            "message": "Argument of an invalid type int supplied for foreach, only iterables are supported.",
            "severity": "error"
        }
    ]
}
